lib/pages/main/main_vc.dart

```diff
@@ -1,456 +1,282 @@
-    }
-  }
+  Widget _uuidWithClipboard(){
+    return FutureBuilder(
+      future: UserManager.getUUID(),
+      builder: (BuildContext context, AsyncSnapshot<String?> snapshot) {
+        if (snapshot.connectionState == ConnectionState.waiting) {
+          return CircularProgressIndicator();
+        } else if (snapshot.hasError) {
+          return Text('Error: ${snapshot.error}');
+        }
 
-  _createAdultReport() async {
-    var createReport = await Utils.createNewReport('adult');
-    loadingStream.add(false);
-    if (createReport) {
-      await Navigator.push(
-        context,
-        MaterialPageRoute(builder: (context) => AdultReportPage()),
-      );
-    } else {
-      print('Adult report was not created');
-      loadingStream.add(false);
-      await Utils.showAlert(MyLocalizations.of(context, 'app_name'),
-          MyLocalizations.of(context, 'server_down'), context);
-    }
+        print(snapshot.data);
+        return Row(
+          children: [
+            Text(snapshot.data ?? '',
+              style: TextStyle(
+                color: Colors.black.withOpacity(0.7),
+                fontSize: 9
+              )
+            ),
+            GestureDetector(
+              child: Icon(
+                Icons.copy_rounded,
+                size: 18,
+              ),
+              onTap: () {
+                final data = snapshot.data;
+                if (data != null) {
+                  Clipboard.setData(ClipboardData(text: data));
+                  ScaffoldMessenger.of(context).showSnackBar(
+                    SnackBar(
+                      content: Text('Copied to Clipboard'),
+                    ),
+                  );
+                } else {
+                  // Display an error message for troubleshooting
+                  ScaffoldMessenger.of(context).showSnackBar(
+                    SnackBar(
+                      content: Text('Error: Unable to copy to clipboard. Data is null.'),
+                    ),
+                  );
+                }
+              },
+            )
+          ],            
+        );
+      }
+    );
   }
 
-  _createSiteReport() async {
-    var createReport = await Utils.createNewReport('site');
-    loadingStream.add(false);
-    if (createReport) {
-      await Navigator.push(
-        context,
-        MaterialPageRoute(builder: (context) => BreedingReportPage()),
-      );
-    } else {
-      print('Site report was not created');
-      loadingStream.add(false);
-      await Utils.showAlert(MyLocalizations.of(context, 'app_name'),
-          MyLocalizations.of(context, 'server_down'), context);
-    }
+  Widget _buildCustomTile(int index, IconData icon, String title, context){
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 12.0),
+      child: Container(
+        decoration: BoxDecoration(
+          color: _selectedIndex == index ? Color.fromARGB(120, 255, 153, 0) : Colors.transparent,
```

Comment:
Could you try if either of these colors look visually more pleasant than the one used?
- `Colors.orange.shade50`
- `Colors.orange.shade100`

Reply:
shade50 is too light, but shade100 looks like that:
![alt text](image.png)

Reply:
Nice! If shade100 is too light, you can try with shade200. Choose the one better in your opinion :)